packages/w/webcomponents-template-polyfill.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "webcomponents-template-polyfill",
+  "description": "HTML template element polyfill.",
+  "keywords": [
+    "component",
+    "dom",
+    "html5",
+    "ie",
+    "ie9",
+    "ie10",
+    "ie11",
+    "internet explorer",
+    "javascript",
+    "js",
+    "legacy",
+    "polyfill",
+    "runtime",
+    "template",
+    "web"
+  ],
+  "license": "BSD-3-Clause",
+  "homepage": "https://github.com/webcomponents/polyfills/tree/master/packages/template",
+  "repository": {
+    "type": "git",
+    "url": "git+https://webcomponents@github.com/webcomponents/polyfills.git"
+  },
+  "filename": "template.min.js",
+  "autoupdate": {
+    "source": "npm",
+    "target": "@webcomponents/template",
+    "fileMap": [
+      {
+        "basePath": ".",
+        "files": [
+          "*.@(js|ts|map)"
```

```suggestion
          "*.js?(.map)",
          "externs/*.d.ts"
```
It looks like your typings are in a subdirectory, so that'll need to be specified here